poetry.lock

```diff
@@ -16,7 +16,7 @@ python-versions = "*"
 
```

Comment:
Is there any particular reason you updated the dependencies on this branch? It's fine but it doesn't really fit with the theme of documentation tidy-ups.

In general, I wouldn't explicitly run `poetry update` on a branch unless you really need to. If you're just bumping dependencies for the sake of it, let dependabot bump them instead.

Reply:
I added a dependancy (`markdown_include`) because I wanted to try something out and committed the change. Then down the line I decided against using it and removed it. Because I had committed the change I couldn't figure out how to return the poetry lock to its original state. Is there a better way to handle situations like this?

Reply:
You basically have two options:

1. `git revert [commit hash]` to undo the change and make a new commit
2. Rewrite the history in some way to drop that commit altogether (e.g. interactive rebase: https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)

They have slightly different use cases, but I think either would be fine here. `git revert` is generally less of a faff.

Reply:
I'd definitely do `git revert` here. Rewriting the history just to avoid a commit here is pain for little obvious gain 😄

Reply:
Yeah you're probably right :smile: